Using-Mastodon/Apps.md

```diff
@@ -134,5 +134,6 @@ Some people have started working on apps for the Mastodon API. Here is a list of
 |**[Tootmap](https://tootmap.net/)**|Web browser|<https://github.com/wakin-/Tootmap>|[@wakin@biwakodon.com](https://biwakodon.com/@wakin)|
 |**[Mastodon User Matching](http://vinayaka.distsn.org/)**|Web browser|<https://github.com/distsn/vinayaka>|[@hakabahitoyo@2.distsn.org](https://2.distsn.org/@hakabahitoyo)|
 |[mastodon-archive](https://github.com/kensanata/mastodon-backup#mastodon-archive)|CLI|<https://github.com/kensanata/mastodon-backup#mastodon-archive>|[Alex Schroeder](https://alexschroeder.ch/wiki/Contact)|
+|[MastoPurge](https://github.com/ThomasLeister/mastopurge)|CLI|<https://github.com/ThomasLeister/mastopurge>|[Thomas Leister](https://thomas-leister.de/en/about-me/)|
```

Comment:
is mastopurge a "beta" software or a "stable" software? If it's beta it should be italicized, and if it's stable it should be moved above mastodon-archive and bolded. see the leading comment in the file:

> Only list apps and tools that work with every (up-to-date) instance. Mobile apps should provide at least an installable package. Please keep this order: Stable FOSS (title **bold**) > unstable FOSS *(alpha/beta/experimental)* > proprietary/non-FOSS (if your app is FOSS, please provide the source and move it to the FOSS section). Further everything alphabetically (but mobile clients on top, cause that is what most people are looking for)

Reply:
I consider MastoPurge as stable software. Updated the file. Thanks for the hint! :-)